edgecast/waf/access_rule.go

```diff
@@ -37,6 +39,8 @@ type AccessRule struct {
 	// Specify each desired IP address using standard IPv4/IPv6 and CIDR notation.
 	IPAccessControls *AccessControls `json:"ip"`
 
+	MaxFileSize int `json:"max_file_size"`
```

Comment:
I did not realize I missed this property... can you add the following comment to it?
`// MaxFileSize Indicates the maximum file size, in bytes, for a POST request body.`